openedx_webhooks/templates/github_community_pr_comment.md.j2

```diff
@@ -1,52 +1,100 @@
-    [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)
-    for details.
-    {%- endfilter %}
-
-    {{ '\n' }}
-
-    {%- filter replace("\n", " ")|trim %}
-    Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the
-    CLA check by adding a comment here that you have signed it.  If the problem persists, you can tag the `@openedx/cla-problems` team in a
-    comment on your PR for further assistance.
-    {%- endfilter %}
+#### :radio_button: Submit a signed contributor agreement (CLA)
+
+:warning: We ask all contributors to the Open edX project to submit a [signed contributor agreement](https://openedx.org/cla) or indicate their institutional affiliation.
+Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md) file for more information.
+
+If you've signed an agreement in the past, you may need to re-sign.
+See [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/) for details.
+
+Once you've signed the CLA, please allow 1 business day for it to be processed.
+After this time, you can re-run the CLA check by adding a comment below that you have signed it.
+If the CLA check continues to fail, you can tag the `@openedx/cla-problems` team in a comment for further assistance.
 
 <!-- comment:no_cla -->
 {% endif %}
 
+{% if is_first_time %}
+#### :radio_button: Wait for tests to be enabled
+
+It looks like you are contributing to this repository for the first time.
+This means that automated tests won't run automatically, and that you'll need to wait for a test run to be authorized.
+
+If you're experiencing a delay of two weeks or more at this step, tag the [community contributions project managers](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3548807177/Community+Contributions+Project+Manager#Current-OSPR-Project-Managers) in a comment and ask for help.
+{% endif %}
+
+#### :radio_button: Get a green build
+
+If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.
+
+{% if is_draft %}
+#### :radio_button: Update the status of your PR
+
+Your PR is currently marked as a draft.  After completing the steps above, update its status by clicking "Ready for Review", or removing "WIP" from the title, as appropriate.
+
+<!-- comment:end_of_wip -->
+{% endif %}
+
+#### :radio_button: Let us know that your PR is ready for review:
+
+### Who will review my changes?
+
+{% if owner %}
+This repository is currently maintained by `@{{ owner }}`.  Tag them in a comment and let them know that your changes are ready for review.
+{%  else %}
+This repository is currently unmaintained.  To get help with finding a technical reviewer, tag the community contributions project manager for this PR in a comment and let them know that your changes are ready for review:
```

Comment:
I think unmaintained leaves a feeling of not being developed, when the project may still be actively developed but without an owner yet. 
```suggestion
This repository currently has no owner.  To get help with finding a technical reviewer, tag the community contributions project manager for this PR in a comment and let them know that your changes are ready for review:
```

Reply:
I'll let @itsjeyd chime in on this since the text draft comes from him.

Reply:
@DanielVZ96 @xitij2000 

> I'm guessing the idea is that every maintained repo should have an owner.

Yes that's right.

Before the maintenance working group was created and started reforming the concept of maintainership earlier this year, things were a lot fuzzier -- there was the concept of (edX/2U) code ownership, which didn't really include the responsibilities that you'd expect a maintainer to take on, and a separate concept of maintainership (as defined in the context of the [Maintainership Pilot](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3426844690/Maintainership+Pilot) program). For some repos, the two would overlap (i.e., edX/2U officially owned these repos and also signed on as a maintainer in the context of the pilot), and for others they didn't. That was creating confusion and contributing to delays in the OSPR review process.

Of late there's been a tendency to standardize on "maintainer" terminology, so I think I was trying to follow that here and avoid falling back on "code ownership" language.

> ... unmaintained leaves a feeling of not being developed, ...

I do agree with this, though. Since the goal is to eventually get to a point where all [relevant repos](https://docs.google.com/spreadsheets/d/1rMwbRldYwp-xPsAG2vijqwB__9M7P2AgjDxX31G7ucA/edit) are under active maintenance, maybe it would help to meet in the middle and say something like "This repository has no maintainer (yet)."?

Reply:
@xitij2000 Sounds good to me, more nuance would be great 👍 CC @mphilbrick211 ⬆️